src/window.js

```diff
@@ -0,0 +1,30 @@
+// Public: Measure how long a function takes to run.
+//
+// description - A {String} description that will be logged to the console when
+//               the function completes.
+// fn - A {Function} to measure the duration of.
+//
+// Returns the value returned by the given function.
+window.measure = function (description, fn) {
+  let start = Date.now()
```

Comment:
or maybe this whole diffing homegrown function could be replaced by `console.time(description)` and `console.timeEnd(description)`